store/store_test.go

```diff
@@ -108,5 +122,24 @@ func TestGC(t *testing.T) {
 			t.Errorf("alert %v should have been gc'd", alert)
 		}
 	}
-	require.Equal(t, len(resolved), n)
+	// If we manually delete an alert it should be sent back to the callback as it should be considered resolved.
+	// So in total, the callback should have received 3 alerts. 2 that were GC and 1 that was manually deleted.
+	require.Equal(t, 3, ac.Count())
```

Comment:
I would make it more explicit. This probably removes the need of the whole comment.
```suggestion
	require.Equal(t, len(resolved) + len(deleted), ac.Count())
```